tamr_unify_client/attribute/collection.py

```diff
@@ -23,7 +23,7 @@ def by_resource_id(self, resource_id):
         :returns: The specified attribute.
         :rtype: :class:`~tamr_unify_client.attribute.resource.Attribute`
         """
-        return self.by_name(resource_id)
```

Comment:
This is a backwards incompatible change. One idea @nbateshaus and I discussed earlier on is to try the newer API, but if it fails with a `404`, to fallback on the old, less nice way of doing things. This means a user on an older version can still use the latest version of python client to interact with their instance, while letting users with new versions take advantage of the newer APIs.

Let's discuss this during standup.

Reply:
This API has existed since the inception of `versioned/v1` APIs, so it is not a backwards incompatible change.